include/aspect/postprocess/gravity_point_values.h

```diff
@@ -225,32 +224,32 @@ namespace aspect
 
         /**
          * Specify the sampling scheme determining if gravity calculation is performed
-         * for a map of points or a list of points.
+         * for a uniform distribution of points or a list of points.
          */
         enum SamplingScheme
         {
-          map,
-          list
+          uniform_distribution,
+          list_of_points
         } sampling_scheme;
 
         /**
          * Parameter for the list sampling scheme:
-         * List of radius coordinates for the list sampling scheme. Must be in order
-         * with the lists of longitude and latitude.
+         * List of radius coordinates for the list of points sampling scheme.
+         * Must be in order with the lists of longitude and latitude.
```

Comment:
Please rephrase to clarify the meaning (same for the two parameters below):
```suggestion
         * Must follow the same order as the lists of longitude and latitude.
```